test/basic.test.js

```diff
@@ -49,23 +49,25 @@ test("basic style object", () => {
 test("1 level descendant combinator", () => {
   const Test = style("table", {
     color: "red",
-    tr: {
+    "& tr": {
       color: "white"
     }
   })
   expectClassNameAndCssText(
     Test(),
     "p2",
-    ".p2 {color: red;},.p2 tr {color: white;}"
+    ".p2 tr {color: white;},.p2 {color: red;}"
   )
 })
 
+// This works in the browser, but not here. I don't know why
```

Comment:
What is the version of Node.js in your environment? It passed in TravisCI.

Reply:
My version is: v9.11.1. TravisCI doesn't run ```npm run test```, it only runs ```lint```,```tslint``` and ```size-check```